packages/quantic/force-app/main/default/lwc/quanticCaseClassification/quanticCaseClassification.js

```diff
@@ -338,7 +338,26 @@ export default class QuanticCaseClassification extends LightningElement {
    * @returns {void}
    */
   setFieldValue(value) {
-    this.field.update(value);
+    this.field?.update(value);
     this._value = value;
   }
+
+  /**
+   * Get the maximum number of suggestions to be displayed.
+   */
+  @api get maxSuggestions() {
+    return this._maxSuggestions;
+  }
+  /**
+   * Set the maximum number of suggestions to be displayed.
+   * @param {number} value - the value to be set.
+   * @returns {void}
+   */
+  set maxSuggestions(value) {
+    if (Number(value) >= 0) {
```

Comment:
JavaScript is funny that way ;) 
![image](https://user-images.githubusercontent.com/2488155/153265158-9af77630-bb59-4011-9e4c-116221f22b4f.png)

It might be okay lol, this resolves to `0`